autoload/lsp/omni.vim

```diff
@@ -182,14 +184,27 @@ function! lsp#omni#get_vim_completion_item(item, ...) abort
         let l:word = s:remove_typed_part(l:word)
     endif
     let l:kind = lsp#omni#get_kind_text(a:item)
+
+    " add user_data in completion item
+    if g:lsp_text_edit_enabled && has_key(a:item, 'textEdit')
```

Comment:
also would be good to have test case for this.

Reply:
Added 573b3d2b8dc881058c702d931451212708e62d3b.